### PR TITLE
Improe cabin inside camera script

### PR DESCRIPTION
**Pull request by Jean28518**
- Reverts the direction of the mouse wheel zoom inside the cabin. -> That's more familiar with zoom in and zoom out
- Don't hardscript anymore some camera fov settings
- Enable camera movement with mouse only, if right mouse button is clicked.

### CODE REVIEW

File: src/Data/Scripts/Player.gd [hunk 1]
```diff
@@ -691,14 +691,15 @@ func handleCamera(delta: float) -> void:
 		cameraNode.translation = soll_camera_translation
 
 		# FIXME: in the first frame, delta == 0, why?
-		if mouseMotion.length() > 0 and delta > 0:
+		if mouseMotion.length() > 0 and delta > 0 and (Input.is_mouse_button_pressed(BUTTON_RIGHT) or Root.mobile_version):
```
Comment (DasCapschen) [inline src/Data/Scripts/Player.gd:694]: Why the Root.mobile_version ? :thinking:

**Review by DasCapschen — APPROVED** [review]
Looks great, I have no objections 👍🏻

**Review by HaSa1002 — APPROVED** [review]
lgtm

File: src/Data/Scripts/Player.gd (hunk 1 above)
Comment (HaSa1002) [reply src/Data/Scripts/Player.gd:694]: Doesn't make sense. Let's do merge and fix later as it shouldn't break anything.

**Pull request merged**